csrc/device_lower/pass/allocation.cpp

```diff
@@ -1702,7 +1702,8 @@ std::vector<Expr*> insertTMemRegionAllocsAndDeallocs(
                     std::vector<Val*>{
                         IrBuilder::create<kir::TensorIndex>(
                             region->address, expr->fusion()->zeroVal()),
-                        region->num_columns},
+                        GpuLower::current()->commonScalarMap().hoistScalar(
```

Comment:
Why is this `hoistScalar` added?

Reply:
Just to simplify the num_columns expression to avoid generating unnecessary `bitceil` in kernel.